test-integration/scores.coffee

```diff
@@ -0,0 +1,56 @@
+{describe, CourseSelect} = require './helpers'
+selenium = require 'selenium-webdriver'
+{expect} = require 'chai'
+
+TEACHER_USERNAME = 'teacher01'
+
+MAIN_ELEMENTS = [
+  '.scores-report',
+  '.course-scores-wrap',
+  '.nav-tabs',
+  '.export-button',
+  '.course-scores-title',
+  '.course-scores-container'
+]
+
+
+describe 'Student Scores', ->
+
+  @it 'loads HS scores table', ->
+    @login(TEACHER_USERNAME)
+    CourseSelect.goTo(@, 'PHYSICS')
+    @waitClick(css: '.calendar-actions a:nth-child(3)')
+    @sleep(1000)
+
+    elements = MAIN_ELEMENTS
+
+    for element in elements
+      @driver.findElement(css: element)
```

Comment:
I think this might need a `@driver.findElement(css: element).then -> console.log ....`

_Aside:_ ES7's `await` keyword might be easier/concise; I can take a look at adding babel (probably easier for greg too)